gax-java/gax/src/main/java/com/google/api/gax/rpc/ServerStream.java

```diff
@@ -89,6 +91,10 @@ public Iterator<V> iterator() {
     return iterator;
   }
 
+  public Stream<V> stream() {
```

Comment:
Can you please add Javadoc? All public methods are required to have Javadoc.

Reply:
done